src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -49,6 +49,8 @@ final class CheckoutContext implements Context
 
     private ApiClientInterface $addressesClient;
 
+    private ApiClientInterface $shippingMethodClient;
```

Comment:
To have consistent name with other clients:
```suggestion
    private ApiClientInterface $shippingMethodsClient;
```